### ISSUE TITLE
PPA

### ISSUE THREAD

**Issue by mbnoimi** [rootkiwi/an2linuxserver#36]
Hi

I suggest to create PPA for packaging an2 into Ubuntu based distros. Installing the server manually really annoying.

**Comment by rootkiwi** [comment]
Yeah that is a problem, PPA might be a great solution for ubuntu/debian. PyPI maybe also? I'm not familiar with neither at the moment.

**Comment by pataquets** [comment]
Also dropping another option: package as Docker image.
Simply mounting the DBus socket file inside the container would enable container processes to send notifications to host's DBus daemon.

More info: https://github.com/mviereck/x11docker/wiki/How-to-connect-container-to-DBus-from-host#dbus-user-session-daemon
